protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3870,3 +3887,95 @@ message GetReferralFeeStatsResponse {
   // Fee statistics for the given market or asset, and epoch.
   vega.events.v1.FeeStats fee_stats = 1;
 }
+
+// Request to get the current referral program
+message GetCurrentVolumeDiscountProgramRequest {}
+
+// Response containing the current referral program
+message GetCurrentVolumeDiscountProgramResponse {
+  // Volume discount program currently activated on the network.
+  VolumeDiscountProgram current_volume_discount_program = 1;
+}
+
+// Request to get the volume discount statistics for a given epoch.
+message GetVolumeDiscountStatsForEpochRequest {
+  // Optional epoch at which the statistics have been updated. If not set, last
+  // epoch is used.
+  uint64 at_epoch = 1;
+}
+
+// Response containing the volume discount statistics for the given epoch.
+message GetVolumeDiscountStatsForEpochResponse {
+  // Volume discount statistics for the epoch.
+  VolumeDiscountStatsForEpoch stats = 1;
```

Comment:
Does this not need any pagniation? I would've thought the `repeated PartyVolumeDiscountStats stats = 2;` field within would get quite largeif its over all parties.